client/task-list/tasks/Marketing/Marketing.scss

```diff
@@ -17,4 +17,19 @@
 			color: $gray-700;
 		}
 	}
+
+	.woocommerce-plugin-list__title {
+		background: inherit;
+		padding-top: $gap-larger;
+		padding-bottom: $gap;
+
+		h3 {
+			text-transform: uppercase;
+		}
+	}
+
+	.woocommerce-plugin-list__plugin {
+		border-top: none;
+		border-bottom: 1px solid #e0e0e0;
+	}
```

Comment:
I believe the correct place for these css changes is actually in `PluginList.scss`.

Reply:
> I believe the correct place for these css changes is actually in `PluginList.scss`.

But we don't want to apply these changes to other tasks, do we? Here it's nested under `.woocommerce-task-marketing`.

Reply:
Disregard my objection. I was confused about the class structure (which, to be fair, is confusing) in these files.